src/fal/telemetry/telemetry.py

```diff
@@ -362,3 +365,41 @@ def _replace_value(val):
 
     output = {key: _replace_value(value) for key, value in props.items()}
     return output
+
+
+def _clean_args_list(args: List[str]) -> List[str]:
+    ALLOWLIST = [
+        "--disable-logging",
+        "--keyword",
+        "--project-dir",
+        "--profiles-dir",
+        "--defer",
+        "--threads",
+        "--state",
+        "--experimental-flow",
+        "-s",
+        "--select",
+        "-m",
+        "--models",
+        "--model",
+        "--exclude",
+        "--selector",
+        "--all",
+        "--scripts",
+        "--before",
+        "run",
+        "fal",
+        "-v",
+        "--version",
+        "--debug",
+        "flow",
+        "[PATH]",
+    ]
+    REDACTED = "[REDACTED]"
+    output = []
+    for item in args:
+        if item in ALLOWLIST:
```

Comment:
does this come in pairs? `["--state", "/path/to/state"]`, are we not adding the second part on purpose?

Reply:
no, it's just a list of all args and we're redacting everything that's not in `ALLOWLIST`